docs/_includes/generic-content.html

```diff
@@ -0,0 +1,9 @@
+<section class="content_main">
+
+    {% if page.body %}
+    <section>
```

Comment:
Maaaybe this is a `div` since its direct parent is a section with nothing else in it but this element?

Reply:
Eww yeah works for me.